CHANGES.md

```diff
@@ -6,7 +6,10 @@
 
 * Update `tokio-util` dependency to 0.3, `FramedWrite` trait bound is changed. [#365]
 
+* Update `actix-rt` dependency to 1.1. [#374]
```

Comment:
The change looks good! I think it doesn't explain what the bug is, the PR title is more suitable.

Reply:
```suggestion
* Only poll dropped ContextFut if event loop is running. [#374]
```